### DISCUSSION TITLE
Galaxy Brain

### DISCUSSION THREAD

**Discussion by HolisticDev1992** [Schweinepriester/github-profile-achievements#683] [Q&A] [answered]
How do I get the Galaxy brain achievement? any help is accepted :)

**Comment by bkpecho** [comment, marked as answer]
1. Answer at least 2 discussions.
2. Take note, you cannot mark your own answer as the answer to your own question.
3. Answer questions asked by users in discussions and get marked.
4. For example, if you mark this answer as "Mark as Answer", then I'll get a point for Galaxy Brain.

Hope this helps!
